spec.html

```diff
@@ -43383,6 +43414,44 @@ <h1>The Strict Mode of ECMAScript</h1>
   </ul>
 </emu-annex>
 
+<emu-annex id="sec-host-layering-points">
+  <h1>Host Layering Points</h1>
+  <p>See <emu-xref href="#sec-hosts-and-implementations"></emu-xref> for the definition o host.</p>
+  <emu-annex id="sec-host-hooks-summary">
+    <h1>Host Hooks</h1>
+    <p><b>HostEnqueuePromiseJob(...)</b>: See <emu-xref href="#sec-hostenqueuepromisejob"></emu-xref></p>
+    <p><b>HostEnsureCanCompileStrings(...)</b>: See <emu-xref href="#sec-hostensurecancompilestrings"></emu-xref></p>
+    <p><b>HostFinalizeImportMeta(...)</b>: See <emu-xref href="#sec-hostfinalizeimportmeta"></emu-xref></p>
+    <p><b>HostGetImportMetaProperties(...)</b>: See <emu-xref href="#sec-hostgetimportmetaproperties"></emu-xref></p>
+    <p><b>HostHasSourceTextAvailable(...)</b>: See <emu-xref href="#sec-hosthassourcetextavailable"></emu-xref></p>
+    <p><b>HostImportModuleDynamically(...)</b>: See <emu-xref href="#sec-hostimportmoduledynamically"></emu-xref></p>
+    <p><b>HostPromiseRejectionTracker(...)</b>: See <emu-xref href="#sec-host-promise-rejection-tracker"></emu-xref></p>
+    <p><b>HostResolveImportedModule(...)</b>: See <emu-xref href="#sec-hostresolveimportedmodule"></emu-xref></p>
+    <p><b>InitializeHostDefinedRealm(...)</b>: See <emu-xref href="#sec-initializehostdefinedrealm"></emu-xref></p>
+  </emu-annex>
+  <emu-annex id="sec-host-defined-fields-summary">
+    <h1>Host-Defined Fields</h1>
+    <p>[[HostDefined]] on Realm Records: See <emu-xref href="#sec-table-21"></emu-xref></p>
+    <p>[[HostDefined]] on Script Records: See <emu-xref href="#sec-table-script-records"></emu-xref></p>
+    <p>[[HostDefined]] on Module Records: See <emu-xref href="#sec-table-36"></emu-xref></p>
+    <p>[[HostSynchronizesWith]] on Candidate Executions: See <emu-xref href="#sec-table-candidate-execution-records"></emu-xref></p>
+    <p>[[IsHTMLDDA]]: See <emu-xref href="#sec-IsHTMLDDA-internal-slot"></emu-xref></p>
+    <p>[[Prototype]] on the global object: See <emu-xref href="#sec-global-object"></emu-xref></p>
```

Comment:
The entire global object is host-defined, not just [[Prototype]].

Reply:
Ah good catch.